api/types/desktop.go

```diff
@@ -127,6 +127,198 @@ func (s *WindowsDesktopServiceV3) MatchSearch(values []string) bool {
 	return MatchSearch(fieldVals, values, nil)
 }
 
+// DynamicWindowsDesktop represents a Windows desktop host that is automatically discovered by Windows Desktop Service.
+type DynamicWindowsDesktop interface {
+	// ResourceWithLabels provides common resource methods.
+	ResourceWithLabels
+	// GetAddr returns the network address of this host.
+	GetAddr() string
```

Comment:
Why not return `net.Addr`?

Reply:
It's for consistency between `DynamicWindowsDesktop` and `WindowsDesktop`